static/css/cf-theme-overrides.less

```diff
@@ -0,0 +1,236 @@
+   Capital Framework theme overrides
+   ========================================================================== */
+
+
+/* cf-core
+   ========================================================================== */
+
+// Sizing variables
+
+@base-font-size-px:             16px;
+@base-line-height-px:           22px;
+@base-line-height:              unit(@base-line-height-px / @base-font-size-px);
+@mobile-max:                    599px;
+@tablet-min:                    600px;
+
+// Color variables
+
+// body
+@text:                          @black;
+
+// a
+@link-text:                     @pacific;
+@link-underline:                @pacific;
+@link-text-visited:             @teal;
+@link-underline-visited:        @teal;
+@link-text-hover:               @pacific-50;
+@link-underline-hover:          @pacific-50;
+@link-text-active:              @navy;
+@link-underline-active:         @navy;
+
+// thead
+@thead-text:                    @white;
+@thead-bg:                      @darkgray;
+@td-bg:                         @gray-5;
+@td-bg-alt:                     @gray-10;
+
+// input
+@input-bg:                      @white;
+@input-border:                  @gray;
+@input-border-focus:            @pacific;
+@input-placeholder:             @gray;
+
+// .figure__bordered
+@figure__bordered:              @gray-50;
+
+
+/* cf-buttons
+   ========================================================================== */
+
+// Color variables
+
+// .btn
+@btn-text:                      @white;
+@btn-bg:                        @pacific;
+@btn-bg-hover:                  @pacific-80;
+@btn-bg-active:                 @navy-80;
+
+// .btn__secondary
+@btn__secondary-text:           @white;
+@btn__secondary-bg:             @gray;
+@btn__secondary-bg-hover:       @gray-80;
+@btn__secondary-bg-active:      @darkgray;
+
+// .btn__warning
+@btn__warning-text:             @white;
+@btn__warning-bg:               @redorange;
+@btn__warning-bg-hover:         @redorange-80;
+@btn__warning-bg-active:        @dark-redorange;
+
+// .btn__disabled
+@btn__disabled-text:            @gray;
+@btn__disabled-bg:              @gray-20;
+@btn__disabled-outline:         @gray-20;
+
+// Sizing variables
+
+// .btn
+@btn-font-size:                 14px;
+@btn-v-padding:                 8px;
+@btn-v-padding-modifier-ie:     .8;
+@super-btn-font-size:           18px;
+
+
+/* cf-expandables
+   ========================================================================== */
+
+// Color variables
+
+// .expandable
+@expandable-focus:              @black;
+
+// .expandable_label
+@expandable_label-text:         @black;
+
+// .expandable_link
+@expandable_link-text:          @pacific;
+
+// .expandable__padded
+@expandable__padded-bg:         @gray-10;
+@expandable__padded-bg-hover:   @gray-20;
+@expandable__padded-divider:    @gray-50;
+
+// .expandable-group
+@expandable-group_header-text:  @gray;
+@expandable-group_header-bg:    @gray-10;
+@expandable-group-bg: @white;
```

Comment:
Good catch. Fixed in https://github.com/cfpb/cfgov-refresh/commit/6d4c3ba4a051937f5ede387ea6c3f7f6670bd1fc and issue opened in https://github.com/cfpb/generator-cf/issues/62